app/models/report.rb

```diff
@@ -132,14 +135,10 @@ def no_report
     false
   end
 
-  def summaryStatus
-    return _("Failed")   if error?
-    return _("Modified") if changes?
-    _("Success")
-  end
-
-  # puppet report status table column name
   def self.report_status
     "status"
   end
 end
+
+require_dependency 'config_report'
```

Comment:
It is required for dev env, if you call `Report.all` in dev env before ConfigReport was loaded the STI does not work correctly.